examples/ansible_retry/concord.yml

```diff
@@ -28,3 +35,100 @@ flows:
 
         times: 1
         delay: 3
+
+  retryAfterSuspend:
+    # Retry the playbook after suspending for some period of time
+    - try:
+        - task: ansible
+          in:
+            playbook: playbook/hello.yml
+            limit: "${retryFile}"  # default is null, will be a .retry file on retries
+            inventory:
+              local:
+                hosts:
+                  - "127.0.0.1"
+                  - "127.0.0.2"
+                  - "127.0.0.3"
+                vars:
+                  ansible_connection: "local"
+            extraVars:
+              makeItFail: "${makeItFail}"
+      error:
+        - log: "Ansible error: ${lastError.cause}"
+
+        # Save retry file as an attachment
+        - call: copyRetryFile
```

Comment:
I think `saveRetryFile` can be used instead? https://github.com/walmartlabs/concord/blob/master/plugins/tasks/ansible/src/main/java/com/walmartlabs/concord/plugins/ansible/TaskParams.java#L108

Reply:
Sure, makes sense 👍

Reply:
Sorry, I meant the `saveRetry` parameter of the task. It saves the retry file as a process attachment, so you don't have to copy it manually.

Reply:
Hmmm I think we have a bug with that parameter. It's copying`<playbook>.yml` to `_attachments/ansible.retry`, but it _should_ be copying `<playbook>.retry` to `_attachments/<playbook>.retry`

Reply:
Yeah https://github.com/walmartlabs/concord/blob/cc59284f6840f236305342296b2ce38dfae21578/plugins/tasks/ansible/src/main/java/com/walmartlabs/concord/plugins/ansible/AnsibleTask.java#L230 should be
```java
String retryFile = getNameWithoutExtension(playbookName) + ".retry";
```

Reply:
https://github.com/walmartlabs/concord/pull/451 PR for fix. Will update this example afterwards